src/gui/qgslabelinggui.cpp

```diff
@@ -135,8 +135,7 @@ void QgsLabelingGui::updateCalloutWidget( QgsCallout *callout )
 }
 
 QgsLabelingGui::QgsLabelingGui( QgsVectorLayer *layer, QgsMapCanvas *mapCanvas, const QgsPalLayerSettings &layerSettings, QWidget *parent, QgsWkbTypes::GeometryType geomType )
-  : QgsTextFormatWidget( mapCanvas, parent, QgsTextFormatWidget::Labeling )
-  , mLayer( layer )
```

Comment:
Don't you need to keep this line?

Reply:
Ah - thanks, I missed off removing the mLayer member from QgsLabelingGui. It's been moved to the QgsTextFormatWidget base class.

Reply:
Fixed